packages/flutter_tools/test/general.shard/asset_bundle_test.dart

```diff
@@ -148,6 +148,33 @@ name: example''')
       FileSystem: () => testFileSystem,
       ProcessManager: () => FakeProcessManager(<FakeCommand>[]),
     });
+
+    // https://github.com/flutter/flutter/issues/42723
```

Comment:
Do we usually add issues URLs for regression tests?  I feel like these get out of date when the tests change, vs just looking at the PR via source control.

Reply:
I usually do - in this case I also added a comment to make it clearer what I am testing